frontend/src/components/emissionResults/emissionResults.js

```diff
@@ -72,10 +78,46 @@ const EmissionResults = ({
     </div>
   ));
 
+  const handleSave = async (e) => {
```

Comment:
Might be worth passing this whole function as a prop to avoid passing so many props to the component.